### PR TITLE
Some displayed content changes..

### PR DESCRIPTION
**Pull request by NatarajBTI**
Some displayed content changes..
@ocofaigh - Pease review this PR..It's just displayed info changes..

### CODE REVIEW

File: ibm_catalog.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
       {
         "name": "deploy-arch-ibm-slz-mas",
         "label": "Maximo Application Suite",
-        "product_kind": "solution",
+        "Category": "Enterprise applications",
```
Comment (ocofaigh) [inline ibm_catalog.json:6]: `"product_kind": "solution"` is a required field here. Please add it back. If you want to add this category, you need to add `enterprise_app` to the `tags` array below

**Review by ocofaigh — CHANGES_REQUESTED** [review]
I left 1 comment - this PR needs to be reviewed by the content team, not me

**Review by Tejashree-Shiju — APPROVED** [review]
Thank you for creating the references-architecture folder and placing the md file in it.
Changes look good to me.

File: ibm_catalog.json (hunk 1 above)
Comment (NatarajBTI) [reply ibm_catalog.json]: It's added back..& added "Enterprise applications" under tags array..

**Comment by NatarajBTI** [comment]
> I left 1 comment - this PR needs to be reviewed by the content team, not me

yes, Tejashree reviewed and approved..

File: ibm_catalog.json [hunk 2]
```diff
@@ -9,28 +9,44 @@
           "target_terraform",
           "terraform",
           "reference_architecture",
-          "solution"
+          "solution",
+		  "Enterprise applications"
```
Comment (ocofaigh) [inline ibm_catalog.json:13]: please fi the indentation

Comment (ocofaigh) [reply ibm_catalog.json:13]: also this is not a valid tag. Please re-check what I said in https://github.com/terraform-ibm-modules/terraform-ibm-mas/pull/79#discussion_r1587460134

FYI category tags can be got from running `ibmcloud catalog filter options --all`

**Review by ocofaigh — DISMISSED** [review]

**Comment by ocofaigh** [comment]
@NatarajBTI please run `pre-commit run --all-files` to and commit updates so pipeline can pass. Make sure you have set up your local dev env first: https://terraform-ibm-modules.github.io/documentation/#/local-dev-setup

**Pull request closed without merging**